app/controllers/declaration/RegisterPartnersController.scala

```diff
@@ -30,20 +29,37 @@ import models.status.{RenewalSubmitted, _}
 import play.api.mvc.{AnyContent, Request, Result}
 import services.{ProgressService, StatusService}
 import uk.gov.hmrc.play.frontend.auth.AuthContext
-import uk.gov.hmrc.play.frontend.auth.connectors.AuthConnector
 import utils.DeclarationHelper._
-import utils.StatusConstants
+import utils.AuthAction
 import views.html.declaration.register_partners
 
 import scala.concurrent.Future
 
 @Singleton
-class RegisterPartnersController @Inject()(val authConnector: AuthConnector,
+class RegisterPartnersController @Inject()(authAction: AuthAction,
                                            val dataCacheConnector: DataCacheConnector,
                                            implicit val statusService: StatusService,
                                            implicit val progressService: ProgressService
-                                          ) extends BaseController {
-
+                                          ) extends DefaultBaseController {
+  def businessPartnersView(amlsRegistrationNo: Option[String],
```

Comment:
add line above to separate method from controller signature

Reply:
Done

Reply:
Done